silkworm/infra/concurrency/signal_handler.cpp

```diff
@@ -106,12 +106,31 @@ std::atomic_int SignalHandler::sig_code_{0};
 std::atomic_bool SignalHandler::signalled_{false};
 std::function<void(int)> SignalHandler::custom_handler_;
 bool SignalHandler::silent_{false};
-std::map<int, void (*)(int)> previous_signal_handlers;
+
+using SignalHandlerFunc = void (*)(int);
+std::map<int, SignalHandlerFunc> previous_signal_handlers;
+
+static SignalHandlerFunc register_signal_action(const int signal_number, SignalHandlerFunc handler_func) {
```

Comment:
```suggestion
static SignalHandlerFunc register_signal_action(const int sig_code, SignalHandlerFunc handler_func) {
```
so it matches naming in the other parts of the class